src/Microsoft.Data.SqlClient/tests/FunctionalTests/AlwaysEncryptedTests/ExceptionRegisterKeyStoreProvider.cs

```diff
@@ -4,69 +4,131 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
```

Comment:
Please, remove unused `using System.Collections.ObjectModel` from here and `SqlConnection`s.